.github/workflows/nix-linux-cu9.yml

```diff
@@ -6,27 +6,54 @@ on:
       - master
 
 jobs:
-  build:
-
+  tests:
+    runs-on: macOS-latest
```

Comment:
We should use not macOS but ubuntu.

Reply:
It may be the cause of the build-shell-cache crash.

Reply:
thanks, I fixed this, but the issues remained. I pushed my locally built dependencies to cachix now, and this brought down the build time to 11 minutes. It succeeds now it seems.